.github/workflows/bibcop.yml

```
@@ -0,0 +1,19 @@
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  checkmake:
+    timeout-minutes: 15
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: yegor256/bibcop-action@0.0.4
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)